app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -127,14 +172,67 @@ const useStyles = makeStyles((theme) => ({
     alignSelf: "center",
     fontWeight: "bold",
   },
+  menuContainer: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+  },
+  menu: {
+    boxShadow: theme.shadows[1],
+    minWidth: "12rem",
+  },
+  menuPopover: {
+    transform: "translateY(1rem)",
+  },
+  menuItem: {
+    width: "100%",
+  },
+  menuItemMessage: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  notificationCount: {
+    color: grey[500],
+    fontWeight: "bold",
+  },
+  menuBtn: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "center",
+    border: `1px solid ${grey[300]}`,
+    borderRadius: 999,
+    backgroundColor: grey[200],
+    padding: theme.spacing(1),
+    transition: "300ms ease-in-out",
```

Comment:
```suggestion
    transition: `${theme.transitions.duration.short}ms ${theme.transitions.easing.easeInOut}`,
```
Only just noticed this, but we should use the value from the theme directly for more consistency here too

Reply:
Thanks, i didn't know there was transition values in theme. (not learn all the possibilities yet ;) )